### ISSUE TITLE
Go to import NETcdf file dialog if chosen from Open From File Dialog

### ISSUE THREAD

**Issue by dannyparsons** [IDEMSInternational/R-Instat#5209]
A confusion for users I've seen is that they choose a NETcdf (.nc) file from Open From File and then it says it does not recognise the file. They didn't realise there is a separate dialog for NETcdf files and then they think R-Instat can't import them.

To help with this if a user chooses a NETcdf file from the Open From File dialog we should be able to close that dialog and automatically open the NETcdf dialog with the file they selected already chosen.

**Comment by maxwellfundi** [comment]
We have also experienced this before with shape files and I think this would be a quick fix for just one type of file. 
Can we actually get to advance our import from file to detect the file type and use the correct methods to import the files?

**Comment by dannyparsons** [comment]
Yes good to do for other special imports like shape files as well.

Long term it would be good that this is all one dialog and that's the better solution. My suggestion was a quicker fix since you could easily get the import dialog to redirect to the specific dialog when a netcdf file or shape file is chosen and close the main import dialog. Then there's only minimal code changes needed. You could keep the specific dialogs on the menu as well to still be able to do it that way.

But happy if someone wants to work on combining the dialogs instead though. It's not a difficult task, but will just need to be done carefully as the main import dialog has a lot of code in it.

**Comment by maxwellfundi** [comment]
Okay as a quick fix then, I suggest we do a similar thing for other files too then we can embark on getting to this dialog to open all other files.

**Comment by dannyparsons** [comment]
Sounds good, it should just need to add a check on the file type selected and then close and open the right dialog.

**Comment by dannyparsons** [comment]
With the file already selected of course.

**Comment by lilyclements** [comment]
@Ag-Derek can you assign this to someone?

From what I can see, currently, if you try to import a `.nc` file into R-Instat through the general Import dialog, it throws an error saying it's not supported. 

What we want instead is to "get the import dialog to redirect to the specific dialog when a netcdf file or shape file is chosen and close the main import dialog. Then there's only minimal code changes needed."

I have noticed in #5234 that this was implemented but the PR was closed. I'm not sure why it was closed - it is unclear - so I suggest a starting point is to try out the code from this PR and see if that does it -- or what that does do.

There's some sample .nc files to download [here](https://www.unidata.ucar.edu/software/netcdf/examples/files.html) for testing this

**Comment by lilyclements** [comment]
@Ag-Derek have you been able to look into this?
